css/style.css

```diff
@@ -506,3 +524,241 @@ textarea:focus:invalid {
   text-align: center;
   color: rgb(235, 235, 255);
 }
+
+@media screen and (max-width: 992px) {
+  .bootom-shine-1 {
+    position: absolute;
+    width: 150px;
+    height: 150px;
+    left: 600px;
+    filter: blur(0.5px);
+  }
+
+  .bootom-shine-1 > img {
+    position: relative;
+    width: 150px;
+    height: 150px;
+  }
+
+  .chat-container {
+    display: none;
+  }
+
+  .desk-footer {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 992px) {
+  .user-name {
+    margin-left: 156px;
+  }
+
+  .right-nav {
+    display: flex;
+    justify-content: flex-end;
+    margin-right: 156px;
+    width: 50%;
+  }
+
+  .right-nav > .link-list-desk {
+    display: flex;
+    justify-content: space-between;
+    padding-left: 0;
+    list-style: none;
+    width: 50%;
+    height: 100%;
+    align-items: center;
+  }
+
+  li > .menus {
+    text-decoration: none;
+    color: white;
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .link-humburgur {
+    display: none;
+  }
+
+  .top-shine-1 > img {
+    position: absolute;
+    width: 200px;
+    height: 200px;
+    left: 0.5em;
+  }
+
+  .bootom-shine-1 {
+    position: absolute;
+    width: 200px;
+    height: 200px;
+    top: 435px;
+    left: 1100px;
+    filter: blur(0.5px);
+  }
+
+  .bootom-shine-1 > img {
+    position: relative;
+    width: 200px;
+    height: 200px;
+  }
+
+  .experience {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+  }
+
+  .skills > .skill-list {
+    display: flex;
+    flex-direction: column;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 13.1389px;
+    line-height: 25px;
+    color: rgb(255, 255, 255);
+    list-style: none;
+  }
+
+  .social-links {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    justify-content: space-around;
+  }
+
+  .social-links > .social-icon {
+    display: flex;
+    justify-content: space-between;
+    height: 50px;
+    min-width: 20%;
+    align-items: center;
+    padding-left: 0;
+  }
+
+  .details > * {
+    display: none;
+  }
+
+  .work-opening {
+    display: flex;
+    flex-direction: column;
+    margin-left: 100px;
+    margin-right: 100px;
+  }
+
+  .works {
+    display: flex;
+    justify-content: space-between;
+    margin-left: 100px;
+    margin-right: 100px;
+  }
+
+  .project-img {
+    display: flex;
+    flex-direction: column;
+    margin-top: 0;
+  }
+
+  .project-img > img {
+    height: 100%;
+    width: 50vw;
+  }
+
+  .work-grid {
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+  }
+
+  .tech-list {
+    width: 75%;
+  }
+
+  .see-more {
+    width: 40%;
+  }
+
+  .contact-container {
+    display: none;
+  }
+
+  .chat-container {
+    display: flex;
+    justify-content: center;
+    margin-top: 126px;
+  }
```

Comment:
- [ ] [REQUIRED] Your form looks good, the "chat-container class" however, the name `input` seems different. The [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) has the first name and last name on separate inputs but your design has just one input field for the name. Kindly fix this so that your design looks more like the provided template.
- [ ] [REQUIRED] Also note that your `send` button has a different color, even the text has a different color. Please ensure that you apply the necessary padding to the `button` as well. Plus you also need to check the margin between the button and the `text-area`, the margin seems to be smaller compared to the one provided in the figma template.

#### Kindly stick to the figma design as much as possible.

#### Please click each image to have a better view of their differences.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![4](https://user-images.githubusercontent.com/58692590/150568046-ad2fb4d1-2b2b-438d-8c52-86889eeee627.png) | ![44](https://user-images.githubusercontent.com/58692590/150568064-86d519a2-b95b-4c75-a5ad-4248d913f1e8.png)

Reply:
> 

Hi @Whoistolu 

You are right! but from current activity **instruction** says to ignore two input fields and keep one input and to make that**full name**

Reply:
Oh, that's true. Thank you for pointing this out @bushmusi